### ISSUE TITLE
Widen filter dropdown content to fit button with

### ISSUE THREAD

**Issue by bia** [weaveworks/ui-components#434]
An extension of #432, it looks strange when the width of the dropdown container is not the same as the button which pops it out. ~We might also have to choose a minimum width that's a little bit larger than what we have now.~
(let's address possibly narrowing the content in a later issue)
![screenshot-2018-12-03-at-17 14 282](https://user-images.githubusercontent.com/92439/49510332-27192480-f888-11e8-8605-44a3ca27fc70.jpg)
<img width="600" alt="screenshot 2018-12-03 at 17 17 54" src="https://user-images.githubusercontent.com/92439/49510364-42842f80-f888-11e8-831b-2615613a0d86.png">

cc @fbarl

**Comment by foot** [comment]
It does look a little funny if the drop down is smaller than the button. 👍 

The reverse (case 1 above), feels quite natural though I think.

**Comment by bia** [comment]
I agree that case 1 is not as bad as case 2, but I'd rather everything just behave the same way. Plus there's usually quite a bit of empty space towards the center of the search bar to expand the width of the 'Filter' button into.

**Comment by foot** [comment]
We could make the default width a bit larger. But too big and it looks a bit funny too?

I'd still avoid clamping the width of the dropdown items, unless you mean expanding the width of the button to be as wide as the largest item in the list?

**Comment by bia** [comment]
> unless you mean expanding the width of the button to be as wide as the largest item in the list?

Hmmm - sounds complicated. Let's limit this issue to widening the content to fit the width of the button for now and address case 1 later.
